database_auth.py

```diff
@@ -0,0 +1,19 @@
+from mysql.connector import connect 
+from mysql.connector import Error
+
+## change database, user and password as per your configuration.
+
+def connection():
+	try:
+		connection = connect(
+							host = 'localhost',
+							database = '',  ## enter database
```

Comment:
You should use environment variables for managing secrets like user and password